dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -206,3 +206,23 @@ CWPKI2044E.useraction=Ensure that all the certificates in the certificate chain
 CWPKI2045W=CWPKI2045W: The certificate with {0} serial number that is signed by the ACME certificate authority at the {1} URI is not valid until {2}.
 CWPKI2045W.explanation=The valid period on the certificate is in the future. SSL/TLS requests fail until the current date and time are within the range that is specified by the valid period on the certificate.
 CWPKI2045W.useraction=Update the local time on the server if the time is incorrect.
+
+CWPKI2046E=CWPKI2046E: Failed to generate a new account key pair while refreshing the account key pair. The error is ''{0}''.
+CWPKI2046E.explanation=Could not generate a new account key pair.
```

Comment:
Try not to use "could" and explain why a new account key pair was not generated. The useraction suggests that the explanation might be "A new account key pair was not generated because the directory that holds the account key pair file is not writable."

Reply:
I realized this error message is not used. I have commented the lines out and removed the messages.